http.go

```diff
@@ -37,8 +37,16 @@ const WriteWait = 10 * time.Second
 const DefaultExpiry = 7200
 
 // filename sanitization
-// swap specific characters
-var filenameReplacer = strings.NewReplacer("(", "_", ")", "_", "&", "+", "'", ",", "—", "-")
+// swap specific special characters
+var filenameReplacer = strings.NewReplacer(
```

Comment:
Thanks for your interest in the project. Some of the character replacements I'm not comfortable with:
```
" ", "_" - I'd prefer to keep spaces in filenames.
"#", "n" - I think this would be strange e.g #100 would be n100. I'd rather see just 100
```

Reply:
its all up to you, really. Spaces in filenames... scary. feel free to just take what you want and leave the rest, i don't need this pull to be merged but if there is any value for you in there... you are welcome to do whatever (sorry i don't have much time)

Reply:
i normally always use --restrict-filenames when using youtube-dl. I'm 85% running my machines in CLI so spaces really suck